content/tutorials/getting-started/index.md

```diff
@@ -166,60 +175,113 @@ Now, you can directly skip to the [generate a package]({{< ref "#generate-packag
+Select your Simulation class (the only available here) with `1`
 
-Upload it to Gatling Enterprise, either by drag-and-dropping it to the modal, or by clicking on the modal to open the file manager.
+```
+Choose a team from the list:
+Type the number corresponding to your choice and press enter
+[0] <Quit>
+[1] Team 'Default team', id='b236af88-1c24-4623-b872-cb8993651c15'
+```
 
-#### Step 3.3 - Create a Simulation {#create-simulation}
+Select a special team, or the default one for your organization with `1`. This will allow access to the simulation to users within this team.
 
-To get a grasp of how to run a simulation with Gatling, you will find a ready to use script in the bundle. So let's get started with your first simulation!
+```
+Enter a simulation name, or just hit enter to accept the default name (ComputerDatabaseSimulation)
+Waiting for user input...
+```
 
-In the [Simulations section]({{< ref "../../reference/user/simulations/" >}}), click on **Create**.
+Enter the name of your simulation, then hit **Enter**
 
-Choose a name for your simulation and define the team that will have access to this simulation (if none has been created yet, a default one named **Default team** exist).
+```
+Do you want to create a new package or upload your project to an existing one?
+Type the number corresponding to your choice and press enter
+[0] <Quit>
+[1] Create a new package on Gatling Enterprise
+[2] Choose an existing package on Gatling Enterprise
+```
+The package is just the jar file containing your simulation class.
+Choose `1` if you don't have any package yet, to create one. It will be reusable by other simulations later on. 
 
-Under Class name, it is important you enter the name of the script that was uploaded in the package, otherwise the run of your simulation will fail in the next step.
+```
+Enter a package name, or just hit enter to accept the default name (gatling:bundle)
+Waiting for user input...
+```
 
-In this case, the ready-made script called `BasicSimulation.scala` is located in the bundle: `user-files > simulations > computerdatabase`
+Enter the name of your package, then hit **Enter**
 
-Therefore, you need to enter `computerdatabase.BasicSimulation` as the class name.
+```
+Choose the load generators location
+Type the number corresponding to your choice and press enter
+[0] <Quit>
+[1] Location AP - Hong kong
+[2] Location AP - Tokyo
+[3] Location AP Pacific - Mumbai
+[4] Location AP SouthEast - Sydney
+[5] Location Europe - Dublin
+[6] Location Europe - Paris
+[7] Location SA East - São Paulo
+[8] Location US East - N. Virginia
+[9] Location US West - N. California
+[10] Location US West - Oregon
+```
 
-Then, click on the **Next** button.
+Here we configure the Simulation itself.
+Choose the location where you want to spawn your load generators and generate traffic from.  
+For demonstration purposes we will choose the unarguable best city : `6`
```

Comment:
I think you should be as precise as possible, and  unfortunately avoid the jokes :(

Reply:
Nooooooooo